.editorconfig

```diff
@@ -189,6 +189,12 @@ dotnet_diagnostic.IDE0066.severity = suggestion
 # IDE0073: File header
 file_header_template = Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+# IDE0001: Name can be simplified
+dotnet_diagnostic.IDE0001.severity = silent
```

Comment:
📝 This one should not be part of CI for anyone. Its behavior changes frequently and the performance is an order of magnitude worse than the sum of all others.

Reply:
Yes, this is actually not for CI, but shows up as a warning in live analysis as we set entire `Style` category to warning by default. We should consider making the diagnostic descriptor for IDE0001 non-configurable: http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/SimplifyTypeNames/SimplifyTypeNamesDiagnosticAnalyzerBase.cs,41